cypress/integration/ResourcesForResearchers/index.js

```diff
@@ -0,0 +1,264 @@
+  cy.get('.searchbar__container input').should('have.attr', 'placeholder', placeholder);
+});
+
+And('section titled {string} is displayed', (title) => {
+  if (title == "Search For Resources") {
+    cy.get('.r4r-DEFAULT.home-nav__tile > .home-tile__text h2').eq(0).should('contain.text', title);
+  } else if (title == "Browse Resources") {
+    cy.get('.r4r-DEFAULT.home-nav__tile > .home-tile__text h2').eq(1).should('contain.text', title);
+  } else if (title == "Tool Type") {
+    cy.get('.r4r-DEFAULT.home-nav__section').eq(0).should('contain.text', title);
+  } else if (title == "Research Area") {
+    cy.get('.r4r-DEFAULT.home-nav__section').eq(1).should('contain.text', title);
+  }
+});
+
+And('text {string} is displayed', (browseText) => {
+  const regex = new RegExp(browseText);
+  cy.get('.home-nav.r4r-DEFAULT p').invoke('text').should('match', regex);
+
+});
+
+And('{string} is a link', (viewAll) => {
+  const regex = new RegExp(viewAll);
+  cy.get('.r4r-DEFAULT.view-all__link').invoke('text').should('match', regex);
+
+});
+
+And('{string} section displays the following {string} categories', (section, options) => {
+
+  cy.document().then(document => {
+    let label;
+    if (section == "Tool Type") {
+      label = "Browse by Tool Type";
+    } else {
+      label = "Browse by Research Area";
+    };
+    const allAnchorTags = document.querySelectorAll(`[aria-label="${label}"]>div>a`);
+    for (let i = 0; i < allAnchorTags; i++) {
+      cy.get('[aria-label="${label}"]>div>a').eq(i).invoke('text').should('match', options[i]);
+    };
+  })
+
+});
+
+Then('the page options should appear at the bottom of the page', () => {
+  cy.get('.r4r-page-options-container').should('be.visible');
+});
+
+And('the following {string} are displayed', (pageOption) => {
+  const options = document.querySelectorAll(`div#PageOptionsControl1 > ul`);
+  for (let i = 0; i < options; i++) {
+    cy.get('div#PageOptionsControl1 > ul>li>a').should('contain.text', title);
+  }
+});
+
+And("user clicks on {string} link", (viewAll) => {
+  cy.get('.r4r-DEFAULT.view-all__link').click();
+});
+
+And("user is redirected to {string} with {string}", (redirectedPath, urlSearch) => {
+  cy.location('pathname').should('eq', redirectedPath);
+  cy.location('search').should('include', urlSearch);
+});
+
+And("{string} link has a href {string}", (linkText, linkHref) => {
+  cy.get('header > a').contains(linkText).should('have.attr', 'href').and('eq', linkHref);
+});
+
+And("search box is empty", () => {
+  cy.get('.searchbar__container input').should('contain.text', "");
+});
+
+And('results info text is {string}', (resultsInfo) => {
+  const regex = new RegExp(resultsInfo);
+  cy.get('[aria-label="Results count"] p').invoke('text').should('match', regex);
+});
+
+And('both pagers are displayed', () => {
+  cy.get('.r4r-results.r4r-DEFAULT>nav').should('have.length', 2);
+  cy.get('.r4r-results.r4r-DEFAULT>nav').each(($el) => {
```

Comment:
duplicating locators ! Use alias instead https://docs.cypress.io/api/commands/as

or second cy.get can be actually chained off the first like so 

```
 cy.get(".r4r-results.r4r-DEFAULT>nav")
    .should("have.length", 2)
    .each(($el) => {
      expect($el).to.be.visible;
    });
```